### ISSUE TITLE
WARNING dev.to broken, I can no longer blog or edit MAYBE come from this tool?

### ISSUE THREAD

**Issue by Offirmo** [maxime1992/dev-to-git#34]
**EDIT: 2020/12/08 bug fixed, came from dev.to**

After following the blog post instructions, I managed to use dev-to-git to upload a test blog post.

I extended to a second blog post, then dev.to broke. I can no longer blog on dev.to nor edit an existing blog post, always getting 404.

I reported the issue to dev.to https://github.com/forem/forem/issues/11690

At this point, **one can't tell if the bug is from `dev-to-git`, from `dev.to` or from the interaction of them** = I'm not blaming dev-to-git.

But **just in case** I suggest NOT using dev-to-git while the linked bug is not fixed, especially if your dev.to blogposts are critical to your business.

**Comment by maxime1992** [comment]
Hi @Offirmo.

I just made a tiny script to figure out how many repositories are using `dev-to-git` and how many blog posts are being published with it. 

It turns out that `dev-to-git` currently powers 224 blog posts across 88 different repositories :)!

<details>
[  
  {  
    "id": 524905,  
    "relativePathToArticle": "./blog-posts/login-vuex-firebase/login-vuex-firebase.md"  
  },  
  {  
    "id": 533959,  
    "relativePathToArticle": "./blog-posts/roles-vuex-firebase/roles-vuex-firebase.md"  
  },  
  {  
    "id": 527573,  
    "relativePathToArticle": "./blog-posts/hello-world-in-html-css-js/index.md"  
  },  
  {  
    "id": 523572,  
    "relativePathToArticle": "./blog-posts/getting-things-done/getting-things-done.md"  
  },  
  {  
    "id": 526288,  
    "relativePathToArticle": "./blog-posts/deploy-pythonanywhere/deploy-pythonanywhere.md"  
  },  
  {  
    "id": 529442,  
    "relativePathToArticle": "./blog-posts/things-you-need-to-learn-thats-not-writing-code/things-you-need-to-learn-thats-not-writing-code.md"  
  },  
  {  
    "id": 529442,  
    "relativePathToArticle": "./blog-posts/things-you-need-to-learn-thats-not-writing-code/things-you-need-to-learn-thats-not-writing-code.md"  
  },  
  {  
    "id": 502153,  
    "relativePathToArticle": "./blog-posts/first-post-and-its-automated/first-post-and-its-automated.md"  
  },  
  {  
    "id": 490648,  
    "relativePathToArticle": "./blog-posts/3-laravel-with-laradock/3-laravel-with-laradock.md"  
  },  
  {  
    "id": 404268,  
    "relativePathToArticle": "./blog-posts/2-programming-in-linux/2-programming-in-linux.md"  
  },  
  {  
    "id": 373033,  
    "relativePathToArticle": "./blog-posts/1-prestashop-docker-compose/1-prestashop-docker-compose.md"  
  },  
  {  
    "id": 437182,  
    "relativePathToArticle": "./blog-posts/anonymous-coding/anonymous-coding.md"  
  },  
  {  
    "id": 464230,  
    "relativePathToArticle": "./blog-posts/zero-liability-co/zero-liability-co.md"  
  },  
  {  
    "id": 464236,  
    "relativePathToArticle": "./blog-posts/uptime-under-duress/uptime-under-duress.md"  
  },  
  {  
    "id": 464258,  
    "relativePathToArticle": "./blog-posts/dreamcatcher-command/dreamcatcher-command.md"  
  },  
  {  
    "id": 464368,  
    "relativePathToArticle": "./blog-posts/ambient-attribution/ambient-attribution.md"  
  },  
  {  
    "id": 498438,  
    "relativePathToArticle": "./blog-posts/side-effects/side-effects.md"  
  },  
  {  
    "id": 453658,  
    "relativePathToArticle": "./dev.to/how-to-send-github-md-to-dev-to-post/readme.md"  
  },  
  {  
    "id": 142126,  
    "relativePathToArticle": "./blog-posts/about-switchmap-and-friends/about-switchmap-and-friends.md"  
  },  
  {  
    "id": 459027,  
    "relativePathToArticle": "./posts/reach-react-devs/reach-react-devs.md"  
  },  
  {  
    "id": 456154,  
    "relativePathToArticle": "./posts/learn-node/learn-node-00.md"  
  },  
  {  
    "id": 456733,  
    "relativePathToArticle": "./posts/learn-node/learn-node-01.md"  
  },  
  {  
    "id": 463910,  
    "relativePathToArticle": "./posts/learn-node/learn-node-02.md"  
  },  
  {  
    "id": 464750,  
    "relativePathToArticle": "./posts/learn-node/learn-node-03.md"  
  },  
  {  
    "id": 438866,  
    "relativePathToArticle": "./docs/blog/introducing-modern-web/index.md"  
  },  
  {  
    "id": 423014,  
    "relativePathToArticle": "./content/your-intro-to/algorithms/test.md"  
  },  
  {  
    "id": 425857,  
    "relativePathToArticle": "./content/general/host-react-vue-free.md"  
  },  
  {  
    "id": 414170,  
    "relativePathToArticle": "./blog-posts/especificacao-protocolo-para-chat-arquitetura-espaco-de-dados-compartilhado/especificacao-protocolo-para-chat-arquitetura-espaco-de-dados-compartilhado.md"  
  },  
  {  
    "id": 404293,  
    "relativePathToArticle": "./blog-posts/test/test.md"  
  },  
  {  
    "id": 404122,  
    "relativePathToArticle": "./blog-posts/react/react-hooks/react-hooks.md"  
  },  
  {  
    "id": 410075,  
    "relativePathToArticle": "./blog-posts/angular/angular10-upgrade/angularLatestVersionUpgrade.md"  
  },  
  {  
    "id": 438421,  
    "relativePathToArticle": "./blog-posts/typescript/NullishCoalescing.md"  
  },  
  {  
    "id": 439472,  
    "relativePathToArticle": "./blog-posts/typescript/typescript-generics.md"  
  },  
  {  
    "id": 390423,  
    "relativePathToArticle": "./blog-posts/why-i-use-markdown-to-take-notes/why-i-use-markdown-to-take-notes.md"  
  },  
  {  
    "id": 393170,  
    "relativePathToArticle": "./blog-posts/how-do-you-explain-your-excitment-when-you-fix-that-bug/how-do-you-explain-your-excitment-when-you-fix-that-bug.md"  
  },  
  {  
    "id": 397888,  
    "relativePathToArticle": "./blog-posts/how-do-you-make-notes/how-do-you-make-notes.md"  
  },  
  {  
    "id": 368178,  
    "relativePathToArticle": "./blog-posts/so-you-want-to-be-a-developer/so-you-want-to-be-a-developer.md"  
  },  
  {  
    "id": 351005,  
    "relativePathToArticle": "./README.md"  
  },  
  {  
    "id": 332866,  
    "relativePathToArticle": "./blog-posts/nestjs/creacion-de-un-crud-con-api-rest-en-nestjs-4ao3.md"  
  },  
  {  
    "id": 332856,  
    "relativePathToArticle": "./blog-posts/nestjs/calculo-de-la-cobertura-conjunta-unitarios-integracion-5eid.md"  
  },  
  {  
    "id": 350540,  
    "relativePathToArticle": "./blog-posts/stenciljs/empezando-con-stenciljs-35e3.md"  
  },  
  {  
    "id": 350541,  
    "relativePathToArticle": "./blog-posts/stenciljs/integrando-stenciljs-con-storybook-48p1.md"  
  },  
  {  
    "id": 352832,  
    "relativePathToArticle": "./blog-posts/nestjs/integrar-typeorm-migrations-93j.md"  
  },  
  {  
    "id": 353674,  
    "relativePathToArticle": "./blog-posts/cypress/empezando-con-cypress-2egi.md"  
  },  
  {  
    "id": 369482,  
    "relativePathToArticle": "./blog-posts/mongodb/empezando-con-mongodb-442i.md"  
  },  
  {  
    "id": 369518,  
    "relativePathToArticle": "./blog-posts/mongodb/instancia-de-mongodb-en-la-nube-gratis-3cai.md"  
  },  
  {  
    "id": 413186,  
    "relativePathToArticle": "./blog-posts/flutter/empezando-con-flutter-31ob.md"  
  },  
  {  
    "id": 413324,  
    "relativePathToArticle": "./blog-posts/dart/empezando-con-dart-3485.md"  
  },  
  {  
    "id": 415598,  
    "relativePathToArticle": "./blog-posts/rxjs/gestion-del-estado-de-una-spa-solo-con-rxjs-5bni.md"  
  },  
  {  
    "id": 422349,  
    "relativePathToArticle": "./blog-posts/rxjs/implementacion-de-un-crud-generico-con-rxjs-2fko.md"  
  },  
  {  
    "id": 430254,  
    "relativePathToArticle": "./blog-posts/nestjs/creacion-de-un-custom-pipe-para-validar-contra-un-servicio-4bdi.md"  
  },  
  {  
    "id": 452758,  
    "relativePathToArticle": "./blog-posts/nestjs/autenticacion-con-jwt-en-nestjs-147a.md"  
  },  
  {  
    "id": 464901,  
    "relativePathToArticle": "./blog-posts/nestjs/autorizacion-con-roles-en-nestjs-3pkn.md"  
  },  
  {  
    "id": 497504,  
    "relativePathToArticle": "./blog-posts/cypress/ejecutar-test-e2e-en-angular-con-cypress-y-calcular-cobertura-g31.md"  
  },  
  {  
    "id": 325393,  
    "relativePathToArticle": "blog-posts/reason-design-patterns/reason-design-patterns.md"  
  },  
  {  
    "id": 326676,  
    "relativePathToArticle": "blog-posts/updating-old-reasonml-code.md"  
  },  
  {  
    "id": 308258,  
    "relativePathToArticle": "./blog-posts/how-to-find-and-replace-in-vim/how-to-find-and-replace-in-vim.md"  
  },  
  {  
    "id": 308291,  
    "relativePathToArticle": "./blog-posts/how-to-find-and-replace-in-vim-2/how-to-find-and-replace-in-vim-2.md"  
  },  
  {  
    "id": 311656,  
    "relativePathToArticle": "./blog-posts/github-vs-gitlab/github-vs-gitlab.md"  
  },  
  {  
    "id": 312691,  
    "relativePathToArticle": "./blog-posts/vim-100-days-learn-enough-to-get-started/vim-100-days-learn-enough-to-get-started.md"  
  },  
  {  
    "id": 312786,  
    "relativePathToArticle": "./blog-posts/vim-100-days-hjkl/vim-100-days-hjkl.md"  
  },  
  {  
    "id": 303026,  
    "relativePathToArticle": "./blog-posts/2020-04-09-spotify/playing-with-spotify-api-and-auth0.md"  
  },  
  {  
    "id": 308295,  
    "relativePathToArticle": "./blog-posts/2020-04-14-atlassian/auth0-as-atlassian-saml-sso-idp.md"  
  },  
  {  
    "id": 292666,  
    "relativePathToArticle": "./blog-posts/docker-arg-y-variables-de-entorno/docker-arg-y-variables-de-entorno.md"  
  },  
  {  
    "id": 294568,  
    "relativePathToArticle": "./blog-posts/crear-capturas-del-codigo-desde-visual-studio-code-con-codesnap/crear-capturas-del-codigo-desde-visual-studio-code-con-codesnap.md"  
  },  
  {  
    "id": 309953,  
    "relativePathToArticle": "./blog-posts/configurar-yarn-en-tus-proyectos-de-angular/configurar-yarn-en-tus-proyectos-de-angular.md"  
  },  
  {  
    "id": 284563,  
    "relativePathToArticle": "./blog-posts/swift-on-android-building-toolchain/index.md"  
  },  
  {  
    "id": 284596,  
    "relativePathToArticle": "./blog-posts/swiftui-previews-in-modular-app-with-frameworks/index.md"  
  },  
  {  
    "id": 284621,  
    "relativePathToArticle": "./blog-posts/xcodegen-vs-local-swift-packages/index.md"  
  },  
  {  
    "id": 295157,  
    "relativePathToArticle": "./blog-posts/swiftui-in-vst2/index.md"  
  },  
  {  
    "id": 296293,  
    "relativePathToArticle": "./blog-posts/swiftui-preview-in-playground/index.md"  
  },  
  {  
    "id": 300096,  
    "relativePathToArticle": "./blog-posts/swiftui-in-auv3/index.md"  
  },  
  {  
    "id": 274810,  
    "relativePathToArticle": "./blog-posts/posts/post.md"  
  },  
  {  
    "id": 266930,  
    "relativePathToArticle": "./blog-posts/iam-test/iam-test.md"  
  },  
  {  
    "id": 270415,  
    "relativePathToArticle": "./blog-posts/aws-cors/aws-cors.md"  
  },  
  {  
    "id": 298686,  
    "relativePathToArticle": "./blog-posts/go-fundamentals/go-fundamentals.md"  
  },  
  {  
    "id": 306513,  
    "relativePathToArticle": "./blog-posts/react-bootstrap/react-bootstrap.md"  
  },  
  {  
    "id": 263251,  
    "relativePathToArticle": "./blog-posts/git-basic/git-basic.md"  
  },  
  {  
    "id": 254512,  
    "relativePathToArticle": "./blog-posts/hello-world/hello-world.md"  
  },  
  {  
    "id": 258051,  
    "relativePathToArticle": "./blog-posts/understanding-react/understanding-react.md"  
  },  
  {  
    "id": 258452,  
    "relativePathToArticle": "./blog-posts/middleware-on-redux/middleware-on-redux.md"  
  },  
  {  
    "id": 258043,  
    "relativePathToArticle": "./blog-posts/understanding-redux/understanding-redux.md"  
  },  
  {  
    "id": 326911,  
    "relativePathToArticle": "./blog-posts/Learning-about-git/Learning-about-git.md"  
  },  
  {  
    "id": 334083,  
    "relativePathToArticle": "./blog-posts/Deeper-knowledge-about-react-architectural-tips/Deeper-knowledge-about-react-architectural-tips.md"  
  },  
  {  
    "id": 345007,  
    "relativePathToArticle": "./blog-posts/Deeper-knowledge-about-react-jsx-and-lifecycle/Deeper-knowledge-about-react-jsx-and-lifecycle.md"  
  },  
  {  
    "id": 239000,  
    "relativePathToArticle": "./blog-posts/239000/239000.md"  
  },  
  {  
    "id": 239001,  
    "relativePathToArticle": "./blog-posts/239001/239001.md"  
  },  
  {  
    "id": 239620,  
    "relativePathToArticle": "./blog-posts/239620/239620.md"  
  },  
  {  
    "id": 239621,  
    "relativePathToArticle": "./blog-posts/239621/239621.md"  
  },  
  {  
    "id": 239622,  
    "relativePathToArticle": "./blog-posts/239622/239622.md"  
  },  
  {  
    "id": 248997,  
    "relativePathToArticle": "./blog-posts/test-post-for-github/test-post-for-github.md"  
  },  
  {  
    "id": 231289,  
    "relativePathToArticle": "./blog-posts/intellisense-for-cypress-fixture-files/intellisense-for-cypress-fixture-files.md"  
  },  
  {  
    "id": 232092,  
    "relativePathToArticle": "./blog-posts/cypress-cri/cypress-cri.md"  
  },  
  {  
    "id": 233542,  
    "relativePathToArticle": "./blog-posts/private-page/creating-private-page/creating-private-page.md"  
  },  
  {  
    "id": 234173,  
    "relativePathToArticle": "./blog-posts/filter-jest-results/filter-jest-results.md"  
  },  
  {  
    "id": 234172,  
    "relativePathToArticle": "./blog-posts/private-page/eleventy-static-page/eleventy-static-page.md"  
  },  
  {  
    "id": 239589,  
    "relativePathToArticle": "./blog-posts/private-page/travis-ftp/travis-ftp.md"  
  },  
  {  
    "id": 231328,  
    "relativePathToArticle": "./blog-posts/cypress-test-route-not-hit/cypress-test-route-not-hit.md"  
  },  
  {  
    "id": 342174,  
    "relativePathToArticle": "./blog-posts/cypress-native-event/cypress-native-event.md"  
  },  
  {  
    "id": 343642,  
    "relativePathToArticle": "./blog-posts/cypress-file-chooser/cypress-file-chooser.md"  
  },  
  {  
    "id": 345661,  
    "relativePathToArticle": "./blog-posts/get-rid-of-mouse-buttons/get-rid-of-mouse-buttons.md"  
  },  
  {  
    "id": 246735,  
    "relativePathToArticle": "./blog-posts/private-page/fetch-tweets/fetch-tweets.md"  
  },  
  {  
    "id": 248320,  
    "relativePathToArticle": "./blog-posts/private-page/travis-http-php/travis-http-php.md"  
  },  
  {  
    "id": 253153,  
    "relativePathToArticle": "./blog-posts/private-page/eleventy-design/eleventy-design.md"  
  },  
  {  
    "id": 267329,  
    "relativePathToArticle": "./blog-posts/private-page/eleventy-create-pagination/eleventy-create-pagination.md"  
  },  
  {  
    "id": 267330,  
    "relativePathToArticle": "./blog-posts/private-page/eleventy-better-navigation/eleventy-better-navigation.md"  
  },  
  {  
    "id": 262454,  
    "relativePathToArticle": "./blog-posts/comics/herbert-birthday/herbert-birthday.md"  
  },  
  {  
    "id": 265628,  
    "relativePathToArticle": "./blog-posts/share-on-social-media/share-on-social-media.md"  
  },  
  {  
    "id": 270799,  
    "relativePathToArticle": "./blog-posts/analyzing-performance-trap/analyzing-performance-trap.md"  
  },  
  {  
    "id": 275637,  
    "relativePathToArticle": "./blog-posts/pr-11ty-website/pr-11ty-website.md"  
  },  
  {  
    "id": 315233,  
    "relativePathToArticle": "./blog-posts/net-internals/how-to-debug-with-windbg/how-to-debug-with-windbg.md"  
  },  
  {  
    "id": 316495,  
    "relativePathToArticle": "./blog-posts/net-internals/call-c-from-cpp/call-c-from-cpp.md"  
  },  
  {  
    "id": 324236,  
    "relativePathToArticle": "./blog-posts/net-internals/net-bitness/net-bitness.md"  
  },  
  {  
    "id": 354033,  
    "relativePathToArticle": "./blog-posts/net-internals/profiler-fn-enter-leave/profiler-fn-enter-leave.md"  
  },  
  {  
    "id": 397649,  
    "relativePathToArticle": "./blog-posts/net-internals/profiler-fn-enter-leave-x64/profiler-fn-enter-leave-x64.md"  
  },  
  {  
    "id": 411349,  
    "relativePathToArticle": "./blog-posts/net-internals/profiler-fn-enter-arguments/profiler-fn-enter-arguments.md"  
  },  
  {  
    "id": 322762,  
    "relativePathToArticle": "./blog-posts/net-internals/write-net-profiler/write-net-profiler.md"  
  },  
  {  
    "id": 322785,  
    "relativePathToArticle": "./blog-posts/net-internals/how-does-profiler-work/how-does-profiler-work.md"  
  },  
  {  
    "id": 333930,  
    "relativePathToArticle": "./blog-posts/net-internals/debugging-profiler/debugging-profiler.md"  
  },  
  {  
    "id": 335019,  
    "relativePathToArticle": "./blog-posts/net-internals/digging-into-callbacks/digging-into-callbacks.md"  
  },  
  {  
    "id": 348868,  
    "relativePathToArticle": "./blog-posts/net-internals/marshal-example/marshal-example.md"  
  },  
  {  
    "id": 322001,  
    "relativePathToArticle": "./blog-posts/impact-of-branch-predictions/impact-of-branch-predictions.md"  
  },  
  {  
    "id": 326459,  
    "relativePathToArticle": "./blog-posts/ipc-by-named-pipes-cpp-csharp/ipc-by-named-pipes-cpp-csharp.md"  
  },  
  {  
    "id": 249017,  
    "relativePathToArticle": "./blog-posts/cypress-versions/cypress-3.8.3/cypress-3.8.3.md"  
  },  
  {  
    "id": 352645,  
    "relativePathToArticle": "./blog-posts/vuejs-avoid-routes/vuejs-avoid-routes.md"  
  },  
  {  
    "id": 134662,  
    "relativePathToArticle": "./blog-posts/from-a-chemist-to-a-programmer-my-story/from-a-chemist-to-a-programmer-my-story.md"  
  },  
  {  
    "id": 230214,  
    "relativePathToArticle": "./blog-posts/2020/2020.md"  
  },  
  {  
    "id": 221490,  
    "relativePathToArticle": "./blog-posts/name-of-your-blog-post/name-of-your-blog-post.md"  
  },  
  {  
    "id": 226187,  
    "relativePathToArticle": "./blog-posts/prettier/prettier.md"  
  },  
  {  
    "id": 514066,  
    "relativePathToArticle": "./blog-posts/nxtend/ionic-angular/release-1.0.0.md"  
  },  
  {  
    "id": 271244,  
    "relativePathToArticle": "./blog-posts/nxtend/ionic-react/nxtend-ionic-react.md"  
  },  
  {  
    "id": 321827,  
    "relativePathToArticle": "./blog-posts/nxtend/ionic-react/release-2.0.0.md"  
  },  
  {  
    "id": 369339,  
    "relativePathToArticle": "./blog-posts/nxtend/ionic-react/release-2.2.0.md"  
  },  
  {  
    "id": 393175,  
    "relativePathToArticle": "./blog-posts/nxtend/ionic-react/release-3.0.2.md"  
  },  
  {  
    "id": 401364,  
    "relativePathToArticle": "./blog-posts/nxtend/ionic-react/release-3.0.3.md"  
  },  
  {  
    "id": 456201,  
    "relativePathToArticle": "./blog-posts/nxtend/ionic-react/release-3.1.0.md"  
  },  
  {  
    "id": 506663,  
    "relativePathToArticle": "./blog-posts/nxtend/ionic-react/release-4.0.0.md"  
  },  
  {  
    "id": 356073,  
    "relativePathToArticle": "./blog-posts/nxtend/capacitor/build-cross-platform-apps.md"  
  },  
  {  
    "id": 437258,  
    "relativePathToArticle": "./blog-posts/nxtend/capacitor/release-1.1.0.md"  
  },  
  {  
    "id": 506661,  
    "relativePathToArticle": "./blog-posts/nxtend/capacitor/release-2.0.2.md"  
  },  
  {  
    "id": 328276,  
    "relativePathToArticle": "./blog-posts/nx-plugin/adding-deps.md"  
  },  
  {  
    "id": 329242,  
    "relativePathToArticle": "./blog-posts/nx-plugin/ng-add-schematic.md"  
  },  
  {  
    "id": 334326,  
    "relativePathToArticle": "./blog-posts/nx-plugin/external-schematics.md"  
  },  
  {  
    "id": 340125,  
    "relativePathToArticle": "./blog-posts/nx-plugin/extending-schematics.md"  
  },  
  {  
    "id": 346211,  
    "relativePathToArticle": "./blog-posts/nx/github-actions.md"  
  },  
  {  
    "id": 384931,  
    "relativePathToArticle": "./blog-posts/nx/react-cli-tooling.md"  
  },  
  {  
    "id": 462358,  
    "relativePathToArticle": "./blog-posts/nx/default-affected-branch.md"  
  },  
  {  
    "id": 206942,  
    "relativePathToArticle": "./README.md"  
  },  
  {  
    "id": 214298,  
    "relativePathToArticle": "./README.md"  
  },  
  {  
    "id": 211983,  
    "relativePathToArticle": "./blog-posts/customise-your-macos-terminal-experience/customise-your-macos-terminal-experience.md"  
  },  
  {  
    "id": 212163,  
    "relativePathToArticle": "./blog-posts/yet-another-best-macos-apps-list-2019/yet-another-best-macos-apps-list-2019.md"  
  },  
  {  
    "id": 209223,  
    "relativePathToArticle": "./articles/guide-to-gatsby-wordpress-starter-advanced/1-Overview.md"  
  },  
  {  
    "id": 209230,  
    "relativePathToArticle": "./articles/guide-to-gatsby-wordpress-starter-advanced/2-Basic-WordPress-and-Gatsby-Setup.md"  
  },  
  {  
    "id": 211293,  
    "relativePathToArticle": "./articles/guide-to-gatsby-wordpress-starter-advanced/3-Setup-Menu-Navigation.md"  
  },  
  {  
    "id": 211957,  
    "relativePathToArticle": "./articles/guide-to-gatsby-wordpress-starter-advanced/4-Deployment.md"  
  },  
  {  
    "id": 214490,  
    "relativePathToArticle": "./articles/guide-to-gatsby-wordpress-starter-advanced/5-Blog-with-Pagination.md"  
  },  
  {  
    "id": 216573,  
    "relativePathToArticle": "./articles/guide-to-gatsby-wordpress-starter-advanced/6-Images.md"  
  },  
  {  
    "id": 217731,  
    "relativePathToArticle": "./articles/guide-to-gatsby-wordpress-starter-advanced/7-PageBuilder-with-ACF-Flexible-Content.md"  
  },  
  {  
    "id": 202892,  
    "relativePathToArticle": "./Object Orientation in JavaScript - 2019-11-09_1556.md"  
  },  
  {  
    "id": 198439,  
    "relativePathToArticle": "./blog-posts/getting-started-with-hangfire/getting-started-with-hangfire.md"  
  },  
  {  
    "id": 199241,  
    "relativePathToArticle": "./blog-posts/getting-started-with-ansible-windows/getting-started-with-ansible-windows.md"  
  },  
  {  
    "id": 199321,  
    "relativePathToArticle": "./blog-posts/wp-with-wpcli-on-win-subsystem/wp-with-wpcli-on-win-subsystem.md"  
  },  
  {  
    "id": 202884,  
    "relativePathToArticle": "./blog-posts/ansible-provision-dev-win/ansible-provision-dev-win.md"  
  },  
  {  
    "id": 265209,  
    "relativePathToArticle": "./blog-posts/node-api-template/node-api-template.md"  
  },  
  {  
    "id": 197479,  
    "relativePathToArticle": "./blog-posts/repositories-are-not-just-for-db/repositories-are-not-just-for-db.md"  
  },  
  {  
    "id": 177904,  
    "relativePathToArticle": "./blog-posts/production-ready-asp-net-core-rest-api/0-picking-technologies.md"  
  },  
  {  
    "id": 188033,  
    "relativePathToArticle": "./blog-posts/production-ready-asp-net-core-rest-api/1-project-layout.md"  
  },  
  {  
    "id": 181306,  
    "relativePathToArticle": "blog-posts/name-of-your-blog-post/react-hooks.MD"  
  },  
  {  
    "id": 169498,  
    "relativePathToArticle": "./blog-posts/dynamic-options-providers/dynamic-options-providers.md"  
  },  
  {  
    "id": 170454,  
    "relativePathToArticle": "./blog-posts/modules-concepts/modules-concepts.md"  
  },  
  {  
    "id": 173551,  
    "relativePathToArticle": "./blog-posts/generate-knex/generate-knex.md"  
  },  
  {  
    "id": 257583,  
    "relativePathToArticle": "./blog-posts/micro-nats-1/micro-nats-part1.md"  
  },  
  {  
    "id": 257636,  
    "relativePathToArticle": "./blog-posts/micro-nats-2/micro-nats-part2.md"  
  },  
  {  
    "id": 257637,  
    "relativePathToArticle": "./blog-posts/micro-nats-3/micro-nats-part3.md"  
  },  
  {  
    "id": 264292,  
    "relativePathToArticle": "./blog-posts/custom-transporter/custom-transporter-part1.md"  
  },  
  {  
    "id": 264811,  
    "relativePathToArticle": "./blog-posts/custom-transporter/custom-transporter-part2.md"  
  },  
  {  
    "id": 266771,  
    "relativePathToArticle": "./blog-posts/custom-transporter/custom-transporter-part3.md"  
  },  
  {  
    "id": 268875,  
    "relativePathToArticle": "./blog-posts/custom-transporter/custom-transporter-part4.md"  
  },  
  {  
    "id": 274195,  
    "relativePathToArticle": "./blog-posts/custom-transporter/custom-transporter-part5.md"  
  },  
  {  
    "id": 268879,  
    "relativePathToArticle": "./blog-posts/custom-transporter/custom-transporter-part6.md"  
  },  
  {  
    "id": 166880,  
    "relativePathToArticle": "./blog-posts/da-pra-gerar-servico-windows-linux-com-dotnet/article.md"  
  },  
  {  
    "id": 175992,  
    "relativePathToArticle": "./blog-posts/como-monto-meu-ambiente-windows-do-zero/article.md"  
  },  
  {  
    "id": 142660,  
    "relativePathToArticle": "./articles/advanced-nestjs-dynamic-providers/article.md"  
  },  
  {  
    "id": 247435,  
    "relativePathToArticle": "./articles/create-your-first-module-with-deno/article.md"  
  },  
  {  
    "id": 157577,  
    "relativePathToArticle": "./articles/oss/azure-pipelines-for-oss/azure-pipelines-for-oss.md"  
  },  
  {  
    "id": 176242,  
    "relativePathToArticle": "./articles/serverless/your-first-serverless-app/your-first-serverless-app.md"  
  },  
  {  
    "id": 282393,  
    "relativePathToArticle": "./articles/vscode/vm-extension/vm-extension.md"  
  },  
  {  
    "id": 284343,  
    "relativePathToArticle": "./articles/azure/cosmosdb-nestjs/cosmosdb-nestjs.md"  
  },  
  {  
    "id": 295093,  
    "relativePathToArticle": "./articles/vscode/record-tips/record-tips.md"  
  },  
  {  
    "id": 126835,  
    "relativePathToArticle": "./blog-posts/git-workflow-guide.md"  
  },  
  {  
    "id": 152698,  
    "relativePathToArticle": "./blog-posts/152698/my-first-effort-at-dev-to-article-writing-and-git-mesh.md"  
  },  
  {  
    "id": 151330,  
    "relativePathToArticle": "./blog-posts/figma-pop-out/figma-pop-out.md"  
  },  
  {  
    "id": 142126,  
    "relativePathToArticle": "./blog-posts/about-switchmap-and-friends/about-switchmap-and-friends.md"  
  },  
  {  
    "id": 146666,  
    "relativePathToArticle": "./blog-posts/finalize-operator/finalize-operator.md"  
  },  
  {  
    "id": 150191,  
    "relativePathToArticle": "./blog-posts/takeuntil-takewhile/takeWhile-takeUntil-takeWhat.md"  
  },  
  {  
    "id": 157143,  
    "relativePathToArticle": "./blog-posts/from-promises-to-observables/from-promises-to-observables.md"  
  },  
  {  
    "id": 222715,  
    "relativePathToArticle": "./blog-posts/host-listener/host-listener.md"  
  },  
  {  
    "id": 140606,  
    "relativePathToArticle": "./posts/look-postgresql/look-postgresql.md"  
  },  
  {  
    "id": 156322,  
    "relativePathToArticle": "./posts/go-direnv/go-direnv.md"  
  },  
  {  
    "id": 221252,  
    "relativePathToArticle": "./posts/gig/gig.md"  
  },  
  {  
    "id": "140550",  
    "relativePathToArticle": "./blog-posts/test/test.md"  
  },  
  {  
    "id": 154760,  
    "relativePathToArticle": "./blog-posts/using-image-in-gatsby/index.md"  
  },  
  {  
    "id": 139953,  
    "relativePathToArticle": "./blog-posts/thinking-in-gatsby-overview/index.md"  
  },  
  {  
    "id": 135403,  
    "relativePathToArticle": "./blog-posts/First-Post.md"  
  },  
  {  
    "id": 134928,  
    "relativePathToArticle": "./TDDWithURMAndKotlin/part2_increment.md"  
  },  
  {  
    "id": 141826,  
    "relativePathToArticle": "./TDDWithURMAndKotlin/part3_jump.md"  
  },  
  {  
    "id": 170072,  
    "relativePathToArticle": "./TheFunctionChangedMyMind/README-en.md"  
  },  
  {  
    "id": 262839,  
    "relativePathToArticle": "./TalkDSLWithKotlin/README-en.md"  
  },  
  {  
    "id": 134665,  
    "relativePathToArticle": "./blog-posts/continuously-sync-files-with-a-remote-system/article.md"  
  },  
  {  
    "id": 164679,  
    "relativePathToArticle": "./blog-posts/react-native-applying-patches/article.md"  
  },  
  {  
    "id": 134395,  
    "relativePathToArticle": "./blog-posts/my-dev2-git-post-d75/my-dev2-git-post.md"  
  },  
  {  
    "id": 134384,  
    "relativePathToArticle": "./blog-posts/first-blog-post/manage-your-blog-post-with-git-and-travis.md"  
  },  
  {  
    "id": 133928,  
    "relativePathToArticle": "./blog-posts/Hello-world/Hello-World.md"  
  },  
  {  
    "id": 133865,  
    "relativePathToArticle": "./static/creating-ferdaaetlan.md"  
  },  
  {  
    "id": 118907,  
    "relativePathToArticle": "./blog-posts/building-scalable-robusts-and-type-safe-forms-with-angular/building-scalable-robusts-and-type-safe-forms-with-angular.md"  
  },  
  {  
    "id": 133785,  
    "relativePathToArticle": "./blog-posts/manage-dev-to-blog-posts-with-continuous-deployment/manage-dev-to-blog-posts-with-continuous-deployment.md"  
  },  
  {  
    "id": 149676,  
    "relativePathToArticle": "./blog-posts/enigma-part-1/enigma-part-1.md"  
  },  
  {  
    "id": 164254,  
    "relativePathToArticle": "./blog-posts/enigma-part-2/enigma-part-2.md"  
  },  
  {  
    "id": 170056,  
    "relativePathToArticle": "./blog-posts/enigma-part-3/enigma-part-3.md"  
  },  
  {  
    "id": 161221,  
    "relativePathToArticle": "./blog-posts/angular-easily-extract-a-falsy-value-from-an-observable-using-the-async-pipe/angular-easily-extract-a-falsy-value-from-an-observable-using-the-async-pipe.md"  
  },  
  {  
    "id": 313025,  
    "relativePathToArticle": "./blog-posts/building-a-reactive-microwave-for-ryan-cavanaugh-with-rxjs/building-a-reactive-microwave-for-ryan-cavanaugh-with-rxjs.md"  
  },  
  {  
    "id": 428130,  
    "relativePathToArticle": "./blog-posts/implement-a-generic-one-of-type-with-typescript/implement-a-generic-one-of-type-with-typescript.md"  
  },  
  {  
    "id": 133166,  
    "relativePathToArticle": "./blog-posts/name-of-your-blog-post/name-of-your-blog-post.md"  
  },  
  {  
    "id": 133734,  
    "relativePathToArticle": "./blog-posts/building-scalable-robusts-and-type-safe-forms-with-angular/building-scalable-robusts-and-type-safe-forms-with-angular.md"  
  }  
]  
```
</details>

Anyway. Nothing has changed recently on this codebase so unless there's been a breaking change on dev.to's API I don't see how this could "break" dev.to. Your title seems a bit excessive, doesn't it? At least you're the first one reporting this so we'll see if anyone else encounters this in the next few days

> I strongly advise NOT using dev-to-git while the linked bug is not fixed.

`dev-to-git` is a free tool developped on my own time and I don't get paid to maintain it. 
Don't get me wrong, I'd be happy to do so but with a proper issue showing some investigation and explaining how to repro.
Also, you do not **have to** use `dev-to-git` to publish on dev.to. This is just a CLI. Feel free not to use it and write your blog posts directly from dev.to UI

**Comment by Offirmo** [comment]
Hi @maxime1992 

I didn't intend to criticize your awesome tool, I should have stated that I also had suspicions that the problem could come from anywhere, for example the dev.to APIs themselves may be not robust. I edited the initial bug report above to make it clearer.

Once again, I apologize for not having been clear enough when reporting. As my whole dev.to was broken, my intention was only to warn other users.

Anyway, the problem has been fixed, it was coming from dev.to (see linked tickets in https://github.com/forem/forem/issues/11690)
